project-templates/windows/0/project.yml

```diff
@@ -3,3 +3,6 @@ description: Experimental Windows template
 stacks:
   - name: windows
     templateId: 'library:infra*windows'
+  - name: windows-network-services
```

Comment:
Can you see what happens if you launch an existing Rancher setup with a windows setup and then change to a catalog with these changes? I'm not sure what happens to the default environment templates if we edit one that is already released. 

This might need to be removed